src/sass/styles.scss

```diff
@@ -362,7 +362,7 @@ pre {
   margin: 0 0.5em;
 }
 
-.has-error, .has-success {
+.usa-button + .has-error, .usa-button + .has-success {
```

Comment:
What is the use case where any buttons that have errors should disappear?

Reply:
Initially this line was introduced to remove the error/success text that appears under 'Submit' button after submission attempt. Button isn't disappearing in that case.